examples/text_labels_and_annotations/font_table.py

```diff
@@ -75,8 +75,8 @@ def draw_font_table(path):
     # we have indeed selected a Unicode charmap.
     codes = font.get_charmap().items()
 
-    labelc = ["{:X}".format(i) for i in range(16)]
-    labelr = ["{:02X}".format(16 * i) for i in range(16)]
+    labelc = [f"{i:X}" for i in range(16)]
+    labelr = [f"{i:02X}" for i in range(0, 16*16, 16)]
```

Comment:
```suggestion
    labelr = [f"{i * 16:02X}" for i in range(16)]
```
Seems more confusing to do the range with offset?

Reply:
Weirdly, I thought the other way was more confusing.

Reply:
:) sounds good, fine either way with me. This is definitely an improvement as is.